docs/reference/extensions/spring-boot-framework.rst

```diff
@@ -0,0 +1,111 @@
+.. _spring-boot-framework-reference:
+
+spring-boot-framework
```

Comment:
@yanksyoon @medubelko if it's ok with you I think we should focus on the extension itself first, and then do docs on a future PR (I thought that was the plan originally)

Reply:
Sure!

Reply:
I'm fine with this. Should we add this page to the ignore list in the docs conf for now? That way we don't publish unfinished docs.